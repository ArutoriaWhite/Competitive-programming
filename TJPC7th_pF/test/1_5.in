50938898 2
7554654 1
16892812 0
