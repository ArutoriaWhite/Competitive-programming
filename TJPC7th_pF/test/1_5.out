7554654 41600740
